arch/xtensa/src/common/xtensa_user_handler.S

```diff
@@ -193,6 +193,13 @@ _xtensa_user_handler:
 
 	mov		a2, RETVAL							/* a2 = address of new state save area */
 
+	/* Set PS.EXCM to 1 */
+
+	movi	a3, PS_EXCM_MASK
+	l32i	a4, a2, (4 * REG_PS)
+	or	a4, a3, a4
+	s32i	a4, a2, (4 * REG_PS)
```

Comment:
Shouldn't it be called before [`xtensa_user`](https://github.com/apache/nuttx/pull/14672/files#diff-bf4f09df2936401ec123284e73fc661258f077f216afde7bc32d2ba5706ff831R192)?

Reply:
Should be here. I have added explain here: https://github.com/apache/nuttx/pull/14672#issuecomment-2553126081